contributor.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## Development
+
```

Comment:
(Very helpful for new contributors is to check prerequisites. 

this would be ### prerequisites
here you could also add very short sections on 1. how to check if this is there, 2. how to get it

Reply:
Change has been taken care of :)